src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert to DB manufacturer: "
+                    + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturers = connection.createStatement()) {
+            String query = "SELECT * FROM manufacturers WHERE is_deleted = false;";
+            ResultSet resultSet = getAllManufacturers.executeQuery(query);
+            while (resultSet.next()) {
+                Manufacturer manufacturer = createManufacturer(resultSet);
+                manufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all formats from DB", e);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturers WHERE id = ?";
+        try (Connection connecton = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connecton.prepareStatement(query)) {
+            preparedStatement.setObject(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            Manufacturer manufacturer = null;
+            if (resultSet.next()) {
+                manufacturer = createManufacturer(resultSet);
+            }
+            return Optional.of(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer from DB by id:" + id, e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateManufacturer =
+                         connection.prepareStatement(query)) {
+            updateManufacturer.setString(1, manufacturer.getName());
+            updateManufacturer.setString(2, manufacturer.getCountry());
+            updateManufacturer.setLong(3, manufacturer.getId());
+            updateManufacturer.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update in DB manufacturer: "
+                    + manufacturer, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement delete = connection.prepareStatement(query)) {

```

Comment:
```suggestion
                PreparedStatement statement = connection.prepareStatement(query)) {
```
let's use simple names for our statements everywhere because it's obvious what they are